.github/workflows/ci.yaml

```diff
@@ -14,21 +18,141 @@ on:
       - v[0-9]+.[0-9]+
       - cryostat-v[0-9]+.[0-9]+
 
+env:
+  TEST_VERSION: 2.25.0-SNAPSHOT
```

Comment:
I think rather than hardcoding this value, it should just be read from the `pom.xml`. It could even be read from the PR's version of the `pom.xml` in case the PR changed the version string. This could be extracted to a workflow step that uses `xpath` or `mvn help:evaluate` like we have in the [test scripts](https://github.com/cryostatio/cryostat/blob/e9ddcaf6503df2e7bc0ef35cc418d0cb01730c7e/run.sh#L10) to just read the `-core` version string and either dump it into the `GITHUB_ENV` or as a step output.